### ISSUE TITLE
CNN Prepare Results Output

### ISSUE THREAD

**Issue by tiimoS** [literally-a-placeholder/pattern_recognition#23]
• Access to your github so that we can inspect your code.
• Plot showing the accuracy and loss on the training and the validation set with respect to
the training epochs.
• Test accuracy with the best parameters found during validation.
